plugins/board-resources/src/components/AddCard.svelte

```diff
@@ -0,0 +1,155 @@
+<!--
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+// Copyright © 2021 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import type { Card as BoardCard } from '@anticrm/board'
+  import board from '../plugin'
+  import task, { calcRank } from '@anticrm/task'
+  import { AttachedData, generateId, Ref, SortingOrder, Space } from '@anticrm/core'
+  import { IconAdd, Button, TextArea, ActionIcon, IconClose } from '@anticrm/ui'
+  import { translate } from '@anticrm/platform'
+  import { getClient } from '@anticrm/presentation'
+  import { clickOutside } from '../utils/ClickOutside'
+
+  export let space: Ref<Space>
+  export let state: any
+
+  let isOpened = false
+  let title = ''
+  let phTraslate: string = ''
+  let input: TextArea
+
+  const client = getClient()
+  let cardId = generateId() as Ref<BoardCard>
+
+  $: if (!title) {
```

Comment:
It will generate newId if I clear title by hands. is it expected?

Reply:
No, fixed, now it's regenerated only when new card is added.